### DISCUSSION TITLE
Apollo 11: T1, T2, T3 abort times

### DISCUSSION THREAD

**Discussion by jumpjack** [orbiternassp/NASSP#721] [Q&A]
Can anybody help figuring out which were the T1, T2 and T3 times during descent and landing?
I did setup this page to study this topic, but everything is so confused!

https://jumpjack.github.io/Apollo11LEMdata/index-checklists.html

**Comment by indy91** [comment]
Well if you look in the transcript, then you get these times of events:

DOI: 101:36:14.07 (DOI PAD at 100:20:32)
PDI: 102:33:04.36 (PDI PAD at 100:22:05)
T1: Not given to the crew on a PAD, but it's fixed at PDI + 15 min according to Timeline Book and Mission Techniques
T2: 102:54:29.00 (from the T-2 PAD, given to the crew at 100:52:43)
T3: 104:39:41.00 (from the T-3 PAD, given to the crew at 100:52:43)

**Comment by jumpjack** [comment]
Transcripts are a consequence of T1, T2 and T3,  I am looking for the source of T1, T2 and T3: when should they be performed and why?
Links to the PADs?

**Comment by indy91** [comment]
The flight plan has the PAD format: https://history.nasa.gov/afj/ap11fj/a11fp/a11-fp-2-040.jpg That's where they would write down the numbers, although they likely would have done that on the Lunar Surface Data Card in the LM Data Card Book.

Why they should be perform? Well they are times to abort from the lunar surface, so they would have been performed if some issue had occured. T1 immediately after landing, T2 after a more thorough check of all the systems and T3 would be the first "normal" ascent, one orbit of the CSM after the LM has landed.

**Comment by jumpjack** [comment]
It's a total mess :-)

![immagine](https://user-images.githubusercontent.com/1620953/155555485-0515298c-f396-4e1e-9350-bd1933f4b033.png)
